pkg/dependencies/factory.go

```
@@ -266,6 +269,9 @@ func (f *Factory) WithTink(clusterConfigFile string) *Factory {
 		}
 		f.dependencies.Tink = f.executableBuilder.BuildTinkExecutable(tinkerbellDatacenterConfig.Spec.TinkerbellCertURL, tinkerbellDatacenterConfig.Spec.TinkerbellGRPCAuth)
 
+		pbnjClient, _ := pbnj.NewPBNJClient(tinkerbellDatacenterConfig.Spec.TinkerbellPBnJGRPCAuth)
```

why ignore the error here?